src/backend/distributed/transaction/worker_transaction.c

```diff
@@ -117,6 +117,27 @@ SendCommandToWorkersWithMetadata(const char *command)
 }
 
 
+/*
+ * SendCommandListToAllWorkers sends the given command to all workers in
+ * a single transaction.
+ */
+void
+SendCommandListToAllWorkers(List *commandList)
+{
+	ListCell *workerNodeCell = NULL;
+	char *extensionOwner = CitusExtensionOwnerName();
```

Comment:
Note: This is always superuser, so our ownership check in worker_remove_jobdir would not work.